### ISSUE TITLE
Importing macros from a different template file works, but from _self does not

### ISSUE THREAD

**Issue by mjatharvest** [ttempleton/craft-nocache#25]
### Description

I've found that importing a macro using `_self` inside a `nocache` tag throws an error.

For example, if you have `template_example.twig` with a macro defined at the top, then this fails.

````
{% macro rendyEntry(entry) %}
    ...macro code here
{% endmacro %}

{% nocache %}
{% import _self as macros %}
...
````

However, importing them from a different file works as expected.

````
{% nocache %}
{% import "_partials/macros" as macros %}
...
````

### Additional information
- No-Cache version: 2.07
- Craft CMS version: 3.7.7
- PHP version: 7.3.28

Handy plugin! 👍

**Comment by ttempleton** [comment]
Thanks for reporting this, @mjatharvest. In that context `_self` is considered to be the `nocache` block, so it's correct that importing from `_self` didn't work, but it turned out that trying to import from the same template file by specifying the template name (which definitely should work) wasn't working either, so that should now be fixed in No-Cache 2.0.8.

**Comment by mjatharvest** [comment]
Terrific, thanks for clarifying `_self` and for resolving the other issue as well 👍
